### PR TITLE
feature/RWSVIEWER 6  layers to querystrings

### PR DESCRIPTION
**Pull request by petergoes**
- feat(store/map): Add mapboxloaded flag
- feat(LayerListControls): Use v-model for the selectedIds
- deps: Upgrade @deltares components to v0.4.0
- refactor(LayerListControls): Keep track of selected layer ids in store
- feat(layer-helpers): Add getLayersById
- feat: Display layers with ids in url on page load
- feat(Layers): Add selected layers to query params
- feat(router): Preserve layer query params over navigations

### CODE REVIEW

File: src/lib/layer-helpers.js [hunk 1]
```diff
@@ -25,3 +28,10 @@ export const getLayersTags = compose(
   flatten,
   map(props([ 'tags' ])),
 )
+
+export const getLayersById = (layers, ids) => 
+  compose(
+    filter(idIncludedIn(ids)),
+    flatten,
+    pickLayersRecursive,
+  )(layers)
```
Comment (vvandijck) [inline src/lib/layer-helpers.js:37]: missing newline

**Review by vvandijck — APPROVED** [review]
LGTM 👍🏼

File: src/lib/layer-helpers.js (hunk 1 above)
Comment (petergoes) [reply src/lib/layer-helpers.js:37]: don't we have a linting tool to fix that?

Comment (vvandijck) [reply src/lib/layer-helpers.js:37]: We use ESLint, that can/should take care of it. Maybe add the `'eol-last'` rule just to be sure?

**Pull request merged**